easybuild/easyconfigs/o/OpenSMILE/OpenSMILE-3.0.0-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenSMILE'
+version = '3.0.0'
+
+homepage = 'https://github.com/audeering/opensmile'
+description = """openSMILE (open-source Speech and Music Interpretation by Large-space Extraction) is a complete and open-source toolkit for audio analysis, processing and classification especially targeted at speech and music applications, e.g. automatic speech recognition, speaker identification, emotion recognition, or beat tracking and chord detection."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/audeering/opensmile/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['0f7a91276a647c4558211b7a635b8aa41f2418f84b603e76c9b363f848cbbb73']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('CMake', '3.16.1'),
```

Comment:
@levlafayette for this toolchain, we've standardised on `CMake/3.15.3`. Do you really need the newer version?